packages/graphql-language-service-server/src/MessageProcessor.js

```diff
@@ -685,7 +685,8 @@ export function getQueryAndRange(
   if (extname(uri) === '.js') {
     if (
       text.indexOf('graphql`') === -1 &&
-      text.indexOf('graphql.experimental`') === -1
+      text.indexOf('graphql.experimental`') === -1 &&
+      text.indexOf('gql`') === -1
```

Comment:
Instead of having all these special tags, is there some way to make it a config option?

Reply:
I agree this should be a configuration option, however I think this is a good set of defaults. `graphql` is used by relay, and `gql` is used by apollo. If you are ok with merging this PR, I can introduce a separate PR that supports overriding the defaults with configuration

Reply:
Sounds good to me